rasa/shared/nlu/training_data/training_data.py

```diff
@@ -60,6 +60,31 @@ def __init__(
 
         self._fill_response_phrases()
 
+    @staticmethod
+    def _load_lookup_table(lookup_table: Dict[Text, Any]) -> Dict[Text, Any]:
+        """Loads the actual lookup table from file if there is a file specified.
+
+        Checks if the specified lookup table contains a filename in
+        `elements` and replaces it with actual elements from the file.
+        Returns the unchanged lookup table otherwise.
+
+        Params:
+            lookup_table: A lookup table.
+
+        Returns:
+            Updated lookup table where filenames are replaced with the contents of these files.
+        """
+        if Path(lookup_table["elements"]).is_file():
```

Comment:
wouldn't it be like that for Markdown?
```suggestion
        if Path(lookup_table["elements"][0]).is_file():
```

Reply:
That's right. Updated it.